docs/reference/api/client.txt

```diff
@@ -0,0 +1,142 @@
+.. _client:
+
+################################
+ determined.experimental.client
+################################
+
+..
+   automodule: determined.experimental.client
+
+***********
+ ``Login``
+***********
+
+.. automodule:: determined.experimental.client
+   :members: login
+   :member-order: bysource
+
+***********************
+ ``Create Experiment``
+***********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: create_experiment
+   :member-order: bysource
+
+********************
+ ``Get Experiment``
+********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_experiment
+   :member-order: bysource
+
+***************
+ ``Get Trial``
+***************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_trial
+   :member-order: bysource
+
+********************
+ ``Get Checkpoint``
+********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_checkpoint
+   :member-order: bysource
+
+******************
+ ``Create Model``
+******************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: create_model
+   :member-order: bysource
+
+***************
+ ``Get Model``
+***************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_model
+   :member-order: bysource
+
+****************
+ ``Get Models``
+****************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_models
+   :member-order: bysource
```

Comment:
I believe all of these lines could just be something like:

```rest
************
 ``Client``
************

.. automodule:: determined.experimental.client
   :members:
   :member-order: bysource
```

I don't see a lot of value in exposing each function via a separate automodule section.

Reply:
If you do this, then each function won't have it's own place to click on and won't be exposed at the top-level Python API docs under `determined.experimental.client`. See the attached pic.
<img width="234" alt="Screen Shot 2021-06-09 at 1 28 29 PM" src="https://user-images.githubusercontent.com/65423210/121424892-af2f8780-c926-11eb-91c9-b3481f537a44.png">

Reply:
That doesn't sound like a big deal to me.